src/main/java/com/amida/saraswati/jdbcsink/utils/Utils.java

```diff
@@ -0,0 +1,24 @@
+package com.amida.saraswati.jdbcsink.utils;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class Utils {
+
+	public static String[] convertListToArray(List<String> input) {
+
+		String[] output = input.stream().toArray(String[]::new);
+
+		return output;
+	}
+	public static Set<String> convertListToSet(List<String> input) {
+		
+		Set<String> output = new HashSet<String>();
+
```

Comment:
HashSet has a constructor that accepts collection I believe. There is also a stream version of it similar to the one you used above for list (input.stream().collect(Collectors.toSet());). And HashSet has an addAll method. All of which are better than the for loop